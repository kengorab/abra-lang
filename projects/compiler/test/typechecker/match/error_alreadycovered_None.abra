val arr = [1, 2, 3]

match arr[0] {
  None => {}
  None => {}
}