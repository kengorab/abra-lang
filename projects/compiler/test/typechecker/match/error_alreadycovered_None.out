Error at %FILE_NAME%:5:3
Unreachable match case
  |    None => {}
       ^
This case has already been covered by a previous case
